frontend/src/components/App.js

```diff
@@ -6,6 +6,7 @@ import { CopCard, InnerCopCard } from "./Cards/Cards";
 import { ScrollCarousel } from "./Carousel/ScrollCarousel";
 import { InnerCopNav } from "./Navigation/InnerCopNav";
 import { HeaderNav } from "./Navigation/HeaderNav";
+import Footer from "./Navigation/_Footer";
```

Comment:
This import line does not match the above pattern. Let's rename _Footer to FooterNav, since the underscore can be repurposed as a naming convention in the future if we need it.